neurokit2/ecg/ecg_invert.py

```diff
@@ -0,0 +1,72 @@
+import numpy as np
+
+from .ecg_clean import ecg_clean
+
+
+def ecg_invert(ecg_signal, sampling_rate=1000, check_inverted=True):
+    """**ECG signal inversion**
+
+    Checks whether an ECG signal is inverted, and if so, corrects for this inversion.
+
+    Parameters
+    ----------
+    ecg_signal : Union[list, np.array, pd.Series]
+        The raw ECG channel.
+    sampling_rate : int
+        The sampling frequency of ``ecg_signal`` (in Hz, i.e., samples/second). Defaults to 1000.
+    check_inverted : bool, optional
+        Whether to check whether the signal is inverted before inverting it. The default is True.
+        If False, always returns the inverted input signal regardless of whether the input was inverted.
+    Returns
+    -------
+    array
```

Comment:
perhaps we could consistently return a tuple, 1) the returning signal and 2) a bool of whether inversion was performed or not (i.e., whether it was detected as needed).

Then we could replace the check_inverted arg by `force=False` to force the inversion, in which case the return would be `(inverted_ecg, True)`